tox.ini

```diff
@@ -6,7 +6,7 @@ isolated_build = True
 [pytest]
 filterwarnings =
     ignore::DeprecationWarning
-addopts= -n auto
+addopts= -n 1
```

Comment:
Why -n 1?

Reply:
heh it lasted just one second before i rolled it back. for some reason `auto` crashes on my machine